__tests__/app.test.js

```diff
@@ -1,64 +1,157 @@
 require('dotenv').config();
+// const { execSync } = require('child_process');
+const { mungeLocation } = require('../data/munge-location.js');
+const { mungeTrails } = require('../data/munge-trails.js');
+const { mungeWeather } = require('../data/munge-weather.js');
+const { mungeYelp } = require('../data/munge-yelp.js');
+
+// const fakeRequest = require('supertest');
+// const app = require('../lib/app');
+// const client = require('../lib/client');
+
+// describe('app routes', () => {
+//   describe('routes', () => {
+//     let token;
+
+//     beforeAll(async done => {
+//       execSync('npm run setup-db');
+
+//       client.connect();
+
+//       const signInData = await fakeRequest(app)
+//         .post('/auth/signup')
+//         .send({
+//           email: 'jon@user.com',
+//           password: '1234'
+//         });
+
+//       token = signInData.body.token;
+
+//       return done();
+//     });
+
+//     afterAll(done => {
+//       return client.end(done);
+//     });
+
+test('returns location', async () => {
+
+  const expectation =
+  {
+    formatted_query: 'Sibay, городской округ Сибай, Bashkortostan, Volga Federal District, UNDEFINED, Russia',
+    latitude: '52.7206093188075',
+    longitude: '58.6657536770277',
+  };
+  const input = [
+    {
+      display_name: 'Sibay, городской округ Сибай, Bashkortostan, Volga Federal District, UNDEFINED, Russia',
+      lat: '52.7206093188075',
+      lon: '58.6657536770277',
+    }];
+
+
+  const output = mungeLocation(input);
+
+  expect(output).toEqual(expectation);
+});
+test('returns trail', async () => {
```

Comment:
I know you already know this but you'll want to use accurate dummy data. This works but the test isn't terribly accurate at the moment (other than the shape of the data).